packages/elements-core/src/components/MarkdownViewer/CustomComponents/CodeComponent.tsx

```diff
@@ -73,7 +75,9 @@ export const CodeComponent: CustomComponentMapping['code'] = props => {
     }
 
     return (
-      <TryIt httpOperation={isHttpOperation(parsedValue) ? parsedValue : parseHttpRequest(parsedValue)} embeddedInMd />
+      <PersistenceContextProvider>
```

Comment:
Pretty sure this should only be around the embedded try it though right?

Reply:
ah I see the issue you prob ran into, the state is accessed up top in try it component, perhaps just an if/else here? can refactor this whole component in a later pr as we were discussing

Reply:
What do you mean?

This is only used for embedding in markdown - see the `embeddedInMd` prop being set to true. 🤔

Reply:
Oh my bad, nvm!

Reply:
This code is part of `MarkdownViewer/CustomComponents` folder, where we store our overrides for markdown components.